docs/crafted-updates.org

```diff
@@ -0,0 +1,93 @@
+* Crafted Emacs Updates Module
+
+** Installation
+
+To use this module, simply require them in your =init.el= at the appropriate
+points.
+
+#+begin_src emacs-lisp
+;; No additional packages are installed by this module.
+
+;; install the packages
+(package-install-selected-packages :noconfirm)
+
+;; Load crafted-updates configuration
+(require 'crafted-updates-config)
+#+end_src
+
+** Description
+
+The =crafted-updates= module provides functionality to check for updates of
+Crafted Emacs. To do so, it relies on [[https://git-scm.com/][git]]. It will only work if you cloned the
+Crafted Emacs [[https://github.com/SystemCrafters/crafted-emacs][github repository]] to your machine.
+
+If you installed Crafted Emacs by other means, it's probably a good idea not to
+load this module, but take care of updates manually.
+
+*** Usage and Customization
+
+You can use the functionality provided by this module in three ways:
+
+1. On startup
```

Comment:
I'm not sure on this mini "table of contents" when the content is directly following.

Especially in info-buffer, it looks a bit funny given 4-star headings are turned into numbered entries:
```
1. On startup
2. Regularly
3. Manually

1. On Startup
...
```
I'd personally remove it.

Reply:
Good point. I agree.
Would you just remove it or turn it into a regular introductory sentence? "You can use the functionality provided by this module in three ways: on startup, regulary and manually."

Reply:
I think putting it into the introductory sentence makes sense.

Reply:
> I think putting it into the introductory sentence makes sense.

Think so, too. Amended it. Thanks for the feedback!